src/v2/controllers/FormController.js

```diff
@@ -123,8 +119,7 @@ export default Controller.extend({
   },
 
   updateAppStateWithNewIdx: function (idxResp) {
```

Comment:
do we even need this func now ? i mean what's the usefulness.

Reply:
yes and we probably shall reuse this function instead of `trigger remediationSuccess` directly.

Reply:
I feel the func name is not relevant and trigger probably won't need to be encapsulated in a function until and unless we are doing more inside that function.

Reply:
Yeah, I'll rename the function name.